app/models/container_image.rb

```diff
@@ -1,10 +1,19 @@
+require 'MiqContainer/MiqContainer'
+
 class ContainerImage < ActiveRecord::Base
```

Comment:
:/ yeah, this will start to perpetuate the problems we have with VMWare logic being spread all over the place in Vms and Hosts.

Probably makes more sense to create a `ManageIQ::Providers::Kubernetes::ContainerManager::ContainerImage` to contain much of this logic.